src/content/operations/deployment/subctl/_index.en.md

```diff
@@ -250,6 +250,32 @@ When running `benchmark latency`, two types of tests will be executed:
 | `--verbose`                         | Produce verbose logs during benchmark tests
 <!-- markdownlint-enable line-length -->
 
+### `validate`
+
+The `subctl validate` is a diagnostic tool that can be used to validate the Submariner deployment.
+
+This tool can be helpful to detect a specific problem when `subctl verify` failed.
+
+Below is a list of available validate commands:
+
+<!-- markdownlint-disable line-length -->
+| Validate command                    | Description                                                                 | Flags     | Falgs Description
+|:------------------------------------|:----------------------------------------------------------------------------|:----------|:------
+| `validate deployment`               | validate that the Submariner components are properly deployed and running with no overlapping CIDRs
```

Comment:
Since `validate` is getting repeated, shall we remove it and only show the sub-command?